backend/src/routes/v0/auth.py

```
@@ -144,7 +144,7 @@ async def register(
 	}
 )
 async def login(
-	credentials: Annotated[UserLogin, Body()],
+	credentials: UserLogin = Body(default=UserLogin(email="admin@example.com", password="test1234")),
```

Ruff (0.11.9)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)